kernel/userspace.c

```diff
@@ -113,9 +111,9 @@ static size_t obj_size_get(enum k_objects otype)
 	return ret;
 }
 
-static bool node_lessthan(struct rbnode *a, struct rbnode *b)
+static bool node_lessthan(struct rbnode *node_a, struct rbnode *node_b)
 {
-	return a < b;
```

Comment:
Does MISRA have a restriction on too short names?

Reply:
No not really. This was a personal preference. I don't like single letter variables

Reply:
my 5 cents here, we shouldn't mix this kind of thing. I guess is better we have atomic commits fixing specific MISRA-C violation than have one commit fixing all sort of things.
It'll make easier tracking the commit and rule related with MISRA-C.